backend/gke/deploy.sh

```diff
@@ -56,12 +59,11 @@ info "Configuring deployed GKE cluster…"
 ROOTFS=overlay-xfs
 # take first worker node as public ip:
 PUBLIC_IP="$(kubectl get nodes -o json | jq -r '.items[].status.addresses[] | select(.type == "InternalIP").address' | head -n 1)"
-DOMAIN="$PUBLIC_IP.$MAGICDNS"
 if ! kubectl get configmap -n kube-system 2>/dev/null | grep -qi cap-values; then
     kubectl create configmap -n kube-system cap-values \
             --from-literal=garden-rootfs-driver="${ROOTFS}" \
             --from-literal=public-ip="${PUBLIC_IP}" \
```

Comment:
what is the use of public IP? when we will be using lb svc data to create dns

Reply:
I do not believe keeping it for now will hurt. When we know for sure to not need it in this setup we can remove it.

Reply:
No, that is this work. To remove public ip and domain combo with dns entries using lb service.

Reply:
I think there's value on having the ip's so they get added by default on the NodePorts. it doesn't hurt, and we need a public ip anyways for kind & minikube, until we migrate them to something else.